gnucash/report/reports/standard/new-owner-report.scm

```diff
@@ -56,6 +56,7 @@
 (define credit-header (N_ "Credits"))
 (define debit-header (N_ "Debits"))
 (define balance-header (N_ "Balance"))
+(define doclink-header (N_ "Document Links"))
 (define linked-txns-header (N_ "Links"))
```

Comment:
What kind of link are this? If we have "Document Links", it should become "Transaction Links" or whatever it is.

Reply:
These are links from data file to external Uri describing files or web address.

Reply:
doclink-header (N_ "Document Links") is obvisious.
I meant linked-txns-header (N_ "Links")

Reply:
It's the transactions linked to the bill/invoice/voucher/credit note via the "post" operation.

Reply:
So:

Owner Report:
`Document Links` = doclinks
`Transaction Links` vs `Business Links` = invoice->payments and payment->invoices

In Transaction report:
`Enable Links` - remove?